### ISSUE TITLE
Flowjs is broken - identifier `Promise$All`. Could not resolve name

### ISSUE THREAD

**Issue by cristian-sima** [facebook/flow#2548]
Same code as yesterday.

On Windows 10 Pro, 

Flow `0.33.0`

![image](https://cloud.githubusercontent.com/assets/12199042/18988256/784b447a-870e-11e6-9874-81c209e8b16f.png)

**Comment by vkurchatkin** [comment]
This is a problem with `fbjs`, although declaring `Promise$all` for backwards compatibility may be a good idea

**Comment by samwgoldman** [comment]
Oops! We removed `Promise$All` because we added a more general feature. You should be able to add this declaration to a [library definition](https://flowtype.org/docs/third-party.html) for backwards compatibility:

```
type Promise$All = <Elem, T: Iterable<Elem>>(promises: T) => Promise<$TupleMap<T, typeof $await>>;
```

Sorry about the confusion here. I wasn't aware that fbjs was relying on this type.

**Comment by cristian-sima** [comment]
@samwgoldman Any hope that the bug will be fixed in a release today ?

**Comment by JeroenNelen** [comment]
Is this on a roadmap of any kind?

**Comment by cristian-sima** [comment]
@samwgoldman Any tip?

**Comment by samwgoldman** [comment]
Have you tried the workaround I shared above?

**Comment by iamdustan** [comment]
`fbjs` has fixed its usage in `0.8.5` if you can force your version to update.
